dashboard/src/components/AppView/ChartInfo.tsx

```diff
@@ -51,6 +51,20 @@ class ChartInfo extends React.Component<IChartInfoProps> {
     // If update is not set yet we cannot know if there is
     // an update available or not
     if (app.updateInfo) {
+      if (app.updateInfo.error) {
+        return (
+          <div>
+            <AlertTriangle
+              color="white"
+              fill="#FDBA12"
+              className="icon"
+              size={15}
+              style={{ bottom: "-0.2em" }}
+            />{" "}
+            <span>Updates check failed. {app.updateInfo.error.message}</span>
```

Comment:
```suggestion
            <span>Update check failed. {app.updateInfo.error.message}</span>
```

Might make more sense grammatically?